technical-guide/developer/ui.md

```diff
@@ -660,3 +660,84 @@ You can use Storybook's `<Canvas>` element to showcase specific stories to enric
 When including codeblocks, please add code in Clojure syntax (not JSX).
 
 You can find an example MDX file in the [Buttons docs](https://hourly.penpot.dev/storybook/?path=/docs/buttons-docs--docs).
+
+### Replacing and old component
+
+**Create screenshots for visual tests BEFORE making any changes.**
+
+Run the tests in the DS project. On TMUX, stop Shadow and run:
+```bash
+clojure -M:dev:shadow-cljs release main
+```
+This will package the frontend in release mode so the tests run faster.
+
+In your terminal, in the frontend folder, run:
+```bash
+  npx playwright test --ui 
+```
+This will open the test runner UI. Select the DS project and run the tests.
```

Comment:
Let's use the `--project` flag here so we don't need to explain "select the DS project" later.

Also, mind the indentation in the code block.

```zsh
npx playwright text --ui --project=ds
```